clippy_lints/src/blacklisted_name.rs

```diff
@@ -17,33 +17,33 @@ declare_clippy_lint! {
     /// ```rust
     /// let foo = 3.14;
     /// ```
-    pub BLACKLISTED_NAME,
+    pub DISALLOWED_NAME,
     style,
-    "usage of a blacklisted/placeholder name"
+    "usage of a disallowed/placeholder name"
 }
 
 #[derive(Clone, Debug)]
-pub struct BlacklistedName {
-    blacklist: FxHashSet<String>,
+pub struct DisAllowedName {
```

Comment:
I think this should be 
```suggestion
pub struct DisallowedName {
```